app/services/notification/AmazonSimpleEmailServiceAsyncFactory.scala

```diff
@@ -0,0 +1,26 @@
+package services.notification
+
+import javax.inject.Inject
+
+import com.amazonaws.auth.{AWSCredentialsProviderChain, DefaultAWSCredentialsProviderChain}
+import com.amazonaws.auth.profile.ProfileCredentialsProvider
+import com.amazonaws.regions.{Region, Regions}
+import com.amazonaws.services.simpleemail._
+import com.google.inject.Provider
+
+
+class AmazonSimpleEmailServiceAsyncFactory extends Provider[AmazonSimpleEmailServiceAsync]{
+
+  private lazy val provider = new AWSCredentialsProviderChain(
+    new ProfileCredentialsProvider("deployTools"),
+    new DefaultAWSCredentialsProviderChain())
```

Comment:
Maybe better to be explicit here? The `DefaultAWSCredentialsProviderChain` matches loads of stuff, including the local default profile. I've seen people get pretty confused and waste lots of time debugging Janus creds when an app is loading credentials they didn't expect. In this case we want the `InstanceProfileCredentialsProvider` in AWS and to fall back to the Janus profile for local development. We really don't want it to look anywhere else.

```scala
new AWSCredentialsProviderChain(
  new InstanceProfileCredentialsProvider(),
  new ProfileCredentialsProvider("deployTools")
)
```